Update README.md (#1015)
DataDog/integrations-extras: perimeterx/README.md

## Overview

This integration allows [PerimeterX][2] customers to forward their PerimeterX related logs and events to Datadog.

## Setup

All configuration is done by PerimeterX. Please visit the [PerimeterX documentation][3] regarding third party integrations.

### Installation

No installation is required on your host.

### Configuration

> 1. Generate a new Integration API Key in your [Datadog portal][4]

Reviewer: ```suggestion
1. Generate a new Integration API Key in your [Datadog portal][4].
```

> 2. Open a support ticket with [PerimeterX Support][5] and request the Datadog log export integration. Support will need the following information:

Reviewer: ```suggestion
2. Open a support ticket with [PerimeterX Support][5] and request the Datadog log export integration. Support needs the following information:
```

### Metrics

> PerimeterX does not includes metrics for [requests][6].

Reviewer: ```suggestion
PerimeterX does not include metrics for [requests][6].
```
